src/index.tsx

```diff
@@ -1,19 +1,127 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+
+const Counter = (props) => {
+	const [value, setValue] = React.useState(props.initialValue);
+
+	function increment() {
+		setValue(value + 1);
+	}
+
+	function decrement() {
+		setValue(value - 1);
+	}
+
+	return React.createElement(
+		'div',
+		null,
+		React.createElement('span', null, value),
+		React.createElement('button', { onClick: increment }, '+ ADD MOVIE'),
+		React.createElement('button', { onClick: decrement }, '-')
+	);
+};
+
+const SearchForm = (props) => {
+	const [query, setQuery] = React.useState(props.initialQuery);
+
+	function handleInputChange(event) {
+		setQuery(event.target.value);
+	}
+
+	function handleSearch(event) {
+		event.preventDefault();
+		props.onSearch(query);
+	}
+
+	return React.createElement(
+		'form',
+		{ onSubmit: handleSearch },
+		React.createElement('input', {
+			type: 'text',
+			placeholder: 'What do you want to watch...',
+			value: query,
+			onChange: handleInputChange,
+		}),
+		React.createElement(
+			'button',
+			{ type: 'submit', className: 'search-button' },
+			'Search'
+		)
+	);
+};
+
+const GenreSelect = (props) => {
+	function handleSelect(genre) {
+		props.onSelect(genre);
+	}
+
+	const genres = props.genres.map((genre) => {
+		const className = genre === props.selectedGenre ? 'selected' : undefined;
+		return React.createElement(
+			'button',
+			{ key: genre, className: className, onClick: () => handleSelect(genre) },
+			genre
+		);
+	});
+
+	return React.createElement('div', { className: 'genre-buttons' }, genres);
+};
 
 const root = ReactDOM.createRoot(
-  document.getElementById('root') as HTMLElement
+	document.getElementById('root') as HTMLElement
 );
+
 root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+	<div className='container'>
```

Comment:
1) Please replace code in `App` with this piece. Just remove tests for now.
2) Import all the required components
3) Then simply Import `App` here like it was previously